fastparquet/api.py

```diff
@@ -122,36 +122,35 @@ def row_group_filename(self, rg):
         else:
             return self.fn
 
-
     def read_row_group_file(self, rg, columns, categories, index=None,
-                            assign=None):
+                            assign=None, mr_times=[]):
```

Comment:
I suppose map-reduce; the files with this type of data have "created_by: parquet-mr". You would call it int96time (whereas the other thing was interval) ?